packages/Algernon.package/ALGNamedWorldMenuEntry.class/instance/selector.st

```diff
@@ -0,0 +1,4 @@
+accessing
+selector
+	^ (self morph selector
```

Comment:
not every morph understands `#selector`, **imo** we should either rename the instance variable to a more specific type, add a comment here or check whether it understands the message

Reply:
We **know** that morph is a MenuItemMorph, which is easily visible from the code, so I think it's okay here.